package.json

```diff
@@ -35,11 +35,11 @@
   "license": "MIT",
   "repository": "jest-community/jest-extended",
   "devDependencies": {
-    "@types/jest": "^23.3.2",
+    "@types/jest": "^24.0.0",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.0.1",
-    "babel-jest": "^23.0.1",
+    "babel-jest": "^24.0.0",
```

Comment:
?

Reply:
Because I bumped Jest to 24 so I could use `--color`, I also bumped this dependency. Both are dev dependencies.